05.ls/ls.rb

```diff
@@ -59,7 +59,7 @@ def search_files(params, dirpath)
 
 def get_detail_files(file_names, dirpath)
   file_names.map do |file_name|
-    fullpath = "#{dirpath}/#{file_name}"
+    fullpath = File.join(dirpath, '/', file_name)
```

Comment:
```suggestion
    fullpath = File.join(dirpath, file_name)
```

でjoinしてくれるのでは？と思いました〜